.github/workflows/build-docs-optimized.yml

```
@@ -0,0 +1,293 @@
+      artifact-name:
+        description: "The name of the uploaded artifact"
+        value: ${{ jobs.build.outputs.artifact-name }}
+
+jobs:
+  build:
+    name: Build Documentation (Optimized)
+    runs-on: ubuntu-latest
+    outputs:
+      deploy-url: ${{ steps.deployment.outputs.page_url }}
+      version: ${{ steps.calver.outputs.version }}
+      artifact-name: ${{ steps.upload.outputs.artifact-name }}
```

actionlint (1.7.7)

property "artifact-name" is not defined in object type {artifact_id: string}

(expression)